docs/source/eop-docs.rst

```diff
@@ -3,13 +3,76 @@ Ends of Prosody: corppa utilities
 
```

Comment:
A little bit of brief preamble would be helpful context here, could be similar to what's in the readme, maybe something like this:

> Documentation for corppa functionality for working with PPA full-text corpus that may be useful to EoP participants.